test/testlib/client.go

```diff
@@ -181,9 +185,13 @@ func CreateTestWebhookAuthenticator(ctx context.Context, t *testing.T) corev1.Ty
 	createContext, cancel := context.WithTimeout(ctx, time.Minute)
 	defer cancel()
 
+	if webhookSpec == nil {
+		webhookSpec = &testEnv.TestWebhook
+	}
```

Comment:
Rather than this, maybe change all the callers to pass the spec as the param? Almost all the callers are passing `nil` as the param, which makes it harder to understand what that param means when looking at a caller.

Reply:
IE, eliminate the defaulting.  Let them specifically pass `testEnv.TestWebhook` each time.  That seems reasonable.